pycon-india-2015/videos/lightning-talk-deployment-automation-pycon-india-2015.json

```diff
@@ -0,0 +1,15 @@
+{
+    "description": "",
+    "recorded": "",
+    "speakers": [
+        ""
```

Comment:
At https://youtu.be/dEA-0o7kBzY?t=1m51s , I can see `Kunal Aggarwal`. Other talks by "Kunal" seem to be by the same speaker.